.github/workflows/tests.yml

```diff
@@ -111,8 +130,8 @@ jobs:
       - uses: actions/cache@v1
         with:
           path: /home/runner/.composer/cache
-          key: composer-php.${{ matrix.php }}-sylius.${{ matrix.sylius }}.0-${{ github.sha }}
-          restore-keys: composer-php.${{ matrix.php }}-sylius.${{ matrix.sylius }}.0-
+          key: composer2-php:${{ matrix.sylius }}-${{ matrix.php }}-${{ github.sha }}
+          restore-keys: composer2-php:${{ matrix.sylius }}-${{ matrix.php }}-
```

Comment:
If you have a condition of the composer version, shouldn't you have one for the cache as well?